zio-http/src/main/scala/zio/http/ClientConfig.scala

```diff
@@ -0,0 +1,21 @@
+package zio.http
+
+import zio.http.service.ClientSSLHandler.ClientSSLOptions
+import zio.http.socket.SocketApp
+
+case class ClientConfig(
+  socketApp: Option[SocketApp[Any]] = None,
```

Comment:
Let's save it for further PRs, but SocketApp is definitely not configuration for an HTTP client.

Reply:
for now, I left it there.